include/NovelRT/Experimental/Windowing/Glfw/Windowing.Glfw.h

```diff
@@ -10,11 +10,13 @@
 namespace NovelRT::Experimental::Windowing::Glfw
 {
     class GlfwWindowingDevice;
+    class GlfwWindowingPluginProvider;
 }
 
 // Windowing.Glfw dependencies
 #include "../../../Maths/GeoVector2F.h"
 #include "../Windowing.h"
+#include "../../../PluginManagement/PluginManagement.h"
 #include <cstdint>
 
 // TODO: Figure out if this is required
```

Comment:
This wasn't touched in this PR, but did we ever figure this out? We should probably open an issue so that we remember to do it next time :stuck_out_tongue: